docs/sources/operator/_index.md

```diff
@@ -13,7 +13,7 @@ weight: 300
 
 # Static mode Kubernetes operator (Beta)
 
-{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<ALLOY_VERSION>">}}
+{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<next>">}}
```

Comment:
Should be `next` and not `<next>`, right?

Reply:
Oooops... I forgot to push the fix